.github/workflows/build.yml

```diff
@@ -1,10 +1,13 @@
-name: Release
+name: Build Docker Image
```

Comment:
I also renamed this workflow because it had the same as the release.yaml workflow and it was confusing for manual dispatch.